src/types/vue.d.ts

```diff
@@ -14,6 +19,21 @@ declare module 'vue/types/vue' {
     beforeRouteEnter(to: Route, from: Route, next: RouteCallback<any>): any;
     beforeRouteLeave(to: Route, from: Route, next: RouteCallback<any>): any;
     beforeRouteUpdate(to: Route, from: Route, next: RouteCallback<any>): any;
+    $http: AxiosInstance,
+    $vxm: {
+      user: ProxyWatchers & UserStore,
+      mobile: ProxyWatchers & MobileStore,
+      pageData: ProxyWatchers & PageData,
+    }
+  }
+
+  interface VueConstructor {
+    $http: AxiosInstance,
+    $vxm: {
+      user: ProxyWatchers & UserStore,
+      mobile: ProxyWatchers & MobileStore,
+      pageData: ProxyWatchers & PageData,
```

Comment:
I don't see another way, but I'm not a big fan of redeclaring it...
Do you see a way to change the vuex-class-component module to allow for this to happen automatically?
Or perhaps we could import this type from `store.ts`?

Reply:
I don't see any other option since it's defined on the Vue prototype post-hoc, so it can't be statically inferred afaik